File: frontend/stormgames-app/src/app/games/models/game.model.ts
import { IGenre } from "src/app/genres/models/genre.model";

export interface IGame {
    id?: number;
    title: string;
    description: string;
    releaseDate: string;
    developer: string;
    publisher: string;
    genres?: IGenre[];
    genreIds?: Array<number>;
}

File: backend/StormGames.Application/Models/GameModel.cs
using StormGames.Domain.Entities;

namespace StormGames.Application.Models;

public class GameModel
{
    public Guid Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public DateTime ReleaseDate { get; set; }
    public string Developer { get; set; }
    public string Publisher { get; set; }
    public IList<GenreModel>? Genres { get; set; }
}